cime_config/config_component.xml

```diff
@@ -46,7 +46,6 @@
     <default_value>zstar_65L</default_value>
     <values>
       <value grid="oi%tx0.25v1">hycom1</value>
-      <value grid="oi%gx1v6">zstar_60L</value>
```

Comment:
Do you want to drop support for the 60 level vertical grid? I think it would be two steps:

1. remove `zstar_60L` from the `valid_values` list three lines up
2. remove all `zstar_60L` occurrences in `MOM_input.yaml`

Reply:
No reason. Thanks.